CRM/Core/SelectValues.php

```diff
@@ -1189,4 +1189,17 @@ public static function optionValueFields() {
     ];
   }
 
+  /**
+   * Limit-to options for schedule reminders.
+   *
+   * @return array
+   */
+  public static function getLimitToValues() {
+    return [
```

Comment:
it is better to use the more complete declaration here - that is what we are moving towards. `customHtmlType` in the same function uses it